content/en/tracing/setup_overview/open_standards/_index.md

```diff
@@ -93,6 +93,371 @@ service:
+
+##### Receive Traces From Other Containers
+
+- Create a `otel_collector_config.yaml` file. [Here is an example template](https://docs.datadoghq.com/tracing/setup_overview/open_standards/#ingesting-opentelemetry-traces-with-the-collector) to get started. It enables the collector's otlp receiver and datadog exporter.
+
+
+- Configure your application with the appropriate Resource attributes for unified service tagging by adding the metadata [described here](https://docs.datadoghq.com/tracing/setup_overview/open_standards/#opentelemetry-collector-datadog-exporter)
+
+- Create a docker network 
+
+  `docker network create <NETWORK_NAME>`
+  
+- Run the OpenTelemetry Collector container and Application container in the same network. *Note*: When running the application container, ensure the environment variable `OTEL_EXPORTER_OTLP_SPAN_ENDPOINT` is configured to use the appropriate hostname for the Collector. In the example below this would be `opentelemetry-collector`
+ 
+  ```
+  # Datadog Agent
+  docker run -d --name opentelemetry-collector \
+            --network <NETWORK_NAME> \
+            -v /var/run/docker.sock:/var/run/docker.sock:ro \
+            -v /proc/:/host/proc/:ro \
+            -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+            -v $(pwd)/otel_collector_config.yaml:/etc/otel/config.yaml \
+            otel/opentelemetry-collector-contrib:latest
+
+  # Application
+  docker run -d --name app \
+            --network <NETWORK_NAME> \
+            -e OTEL_EXPORTER_OTLP_SPAN_ENDPOINT=http://opentelemetry-collector:55680 \
+            company/app:latest
+  ```
+
+#### Kubernetes
+
+The OpenTelemetry Collector can be run in two types of [deployment scenarios](https://opentelemetry.io/docs/collector/getting-started/#deployment). First, as an OpenTelemetry Collector "agent" running on the same host as the application in a sidecar or daemonset. Second, as a standalone service, e.g. a container or deployment, typically per cluster, datacenter or region.
+
+In order to accurately track the appropriate metadata in Datadog for information and billing purposes, it is recommended the OpenTelemetry Collector be run at least in agent mode on each of the Kubernetes Nodes.
+
+- When deploying the OpenTelemetry Collector as a Daemonset, refer to [the example configuration below](#opentelemetry-kubernetes-example-collector-configuration) as a guide.
+
+- On the application container, use the downward API to pull the host IP; the application container needs an environment variable that points to status.hostIP. The OpenTelemetry Collector container Agent expects this to be named `OTEL_EXPORTER_OTLP_SPAN_ENDPOINT`. Use the [below example snippet](#opentelemetry-kubernetes-example-application-configuration) as a guide.
+
+##### OpenTelemetry Kubernetes Example Collector Configuration
+
+```
+---
+# Give admin rights to the default account
+# so that k8s_tagger can fetch info
+# RBAC Config Here
+---
```

Comment:
This RBAC config should ideally be [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/9f137ea75d249631c411b9d845e83a179032a5b8/processor/k8sprocessor/doc.go#L26-L28) and we should just link to it

Reply:
makes sense

Reply:
Can you open an issue on the contrib repo for this too?